lib/form/js/repeatable.js

```diff
@@ -4,257 +4,338 @@
 (function formRepeatable() {
   'use strict';
 
+  // Disabled one-var and func-names for better readability
```

Comment:
These are disabled in validation.js as well. I will enable and structure them accordingly.